.github/workflows/main.yml

```diff
@@ -0,0 +1,15 @@
+name: Cypress testing
+
+on: [push]
+
```

Comment:
May be worth adding automation whenever something is merged. Not necessary, but I've seen this on other yml files and may be worth considering.